bpython/curtsiesfrontend/interpreter.py

```diff
@@ -1,11 +1,11 @@
 import sys
-from typing import Any, Dict
-
-from pygments.token import Generic, Token, Keyword, Name, Comment, String
+from typing import Any, AnyStr, Dict, List, NoReturn, TextIO, IO, Tuple, Union
+from curtsies.formatstring import fmtstr, FmtStr
+from pygments.token import Generic, Token, Keyword, Name, Comment, String  # type: ignore
 from pygments.token import Error, Literal, Number, Operator, Punctuation
 from pygments.token import Whitespace
-from pygments.formatter import Formatter
-from pygments.lexers import get_lexer_by_name
+from pygments.formatter import Formatter  # type: ignore
```

Comment:
If you install the pygments types (see https://github.com/bpython/bpython/blob/main/.github/workflows/lint.yaml#L42) you won't need to ignore these